server/cmd/mmctl/commands/team_e2e_test.go

```diff
@@ -182,10 +182,12 @@ func (s *MmctlE2ETestSuite) TestModifyTeamsCmdF() {
 		cmd := &cobra.Command{}
 		cmd.Flags().Bool("private", true, "")
 		err := modifyTeamsCmdF(s.th.Client, cmd, []string{teamID})
-		s.Require().NoError(err)
+
+		expectedError := fmt.Sprintf("Unable to modify team '%s' error: You do not have the appropriate permissions.", s.th.BasicTeam.Name)
+		s.Require().Equal(expectedError, err.(*multierror.Error).Unwrap().Error())
```

Comment:
Nit: Do we need to unwarp the error? Maybe `s.Require().ErrorContains()` is an easier way to check for the error message then using `Equal`.

Reply:
Updated. Thank you for the great suggestions.